File: generated-sources/php-slim/mojang-sessions/lib/Model/PlayerProfileTexturePropertyValue.php
<?php
/**
 * PlayerProfileTexturePropertyValue
 */
namespace OpenAPIServer\Model;

/**
 * PlayerProfileTexturePropertyValue
 */
class PlayerProfileTexturePropertyValue
{
    
    /** @var int $timestamp UNIX timestamp in milliseconds*/
    private $timestamp;
    
    /** @var string $profileId The player UUID without hyphens*/
    private $profileId;
    
    /** @var string $profileName The player&#39;s name*/
    private $profileName;
    
    /** @var bool $signatureRequired Only present if unsigned was set to false in the request*/
    private $signatureRequired;
    
    /** @var \OpenAPIServer\Model\PlayerTexture $textures */
    private $textures;
}

File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/PlayerProfileTexturePropertyValue.scala

package org.openapitools.client.model


case class PlayerProfileTexturePropertyValue (
    /* UNIX timestamp in milliseconds */
    _timestamp: Long,
    /* The player UUID without hyphens */
    _profileId: String,
    /* The player's name */
    _profileName: String,
    /* Only present if unsigned was set to false in the request */
    _signatureRequired: Option[Boolean],
    _textures: Option[PlayerTexture]
)
object PlayerProfileTexturePropertyValue {
    def toStringBody(var_timestamp: Object, var_profileId: Object, var_profileName: Object, var_signatureRequired: Object, var_textures: Object) =
        s"""
        | {
        | "timestamp":$var_timestamp,"profileId":$var_profileId,"profileName":$var_profileName,"signatureRequired":$var_signatureRequired,"textures":$var_textures
        | }
        """.stripMargin
}
